modules/core/server/views/layout.server.view.html

```diff
@@ -50,50 +50,50 @@
     <link
       rel="icon"
       type="image/png"
-      href="{{hostPort}}/img/icons/android/512.png"
+      href="{{hostPort}}/placeholder.png"
```

Comment:
I find it a bit confusing if you use one placeholder name (placeholder.png) to reference multiple images. If I would have read the code without the changes, I would assume that all instances of "placeholder.png" should refer to the same image, which would be different from the original code.

Reply:
You are 100% correct, we should ideally have different placeholders or even better a config which allows you to put the full path of the file for each different logo/image.